integration_tests/run_pyspark_from_build.sh

```diff
@@ -57,7 +57,10 @@ else
     else
         AVRO_JARS=$(echo "$SCRIPTPATH"/target/dependency/spark-avro*.jar)
         PARQUET_HADOOP_TESTS=$(echo "$SCRIPTPATH"/target/dependency/parquet-hadoop*.jar)
-        export INCLUDE_PARQUET_HADOOP_TEST_JAR=true
+        REGEX="parquet-hadoop-1\.([0-9]{2})"
+        [[ $PARQUET_HADOOP_TESTS =~ $REGEX  ]]
+        # Make sure we have Parquet version >= 1.12 in the dependency
+        export INCLUDE_PARQUET_HADOOP_TEST_JAR=$([ "${BASH_REMATCH[1]}" -ge "12" ] && echo true || echo false)
```

Comment:
This is going to fail when the Parquet version goes to 2.x or later.  We could leverage `sort -V` to solve the sorting issue, doing something like this:
```suggestion
        MIN_PARQUET_JAR="$SCRIPTPATH/target/dependency/parquet-hadoop-1.12.0-tests.jar"
        # Make sure we have Parquet version >= 1.12 in the dependency
        LOWEST_PARQUET_JAR=$(echo -e "$MIN_PARQUET_JAR\n$PARQUET_HADOOP_TESTS | sort -V | head -1)
        export INCLUDE_PARQUET_HADOOP_TEST_JAR=$([[ "$LOWEST_PARQUET_JAR" == "MIN_PARQUET_JAR" ]] && echo true || echo false)
```

Reply:
Thank you. This is much better and future proof